### PR TITLE
Chapter 17 initial setup

### PR DESCRIPTION
**Pull request by speudusa**
- [x] Reading
- [x] Exercises
- [x] Studio
- [x] Check Your Understanding questions
- [x] Video

### CODE REVIEW

File: content/orm-part-1/exercises/_index.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+title: "Exercises: OMG the ORM!"
+date: 2023-03-10T09:32:46-06:00
+draft: false
+weight: 2
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
```
Comment (codinglikeagirl42) [inline content/orm-part-1/exercises/_index.md:9]:
```suggestion
originalAuthor: Courtney Frey # to be set by page creator
originalAuthorGitHub: speudusa # to be set by page creator
reviewer: Kimberly Horan # to be set by the page reviewer
reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
```

File: content/orm-part-1/next-steps.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Next Steps"
+date: 2023-03-10T09:01:27-06:00
+draft: false
+weight: 4
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
```
Comment (codinglikeagirl42) [inline content/orm-part-1/next-steps.md:9]:
```suggestion
originalAuthor: Courtney Frey # to be set by page creator
originalAuthorGitHub: speudusa # to be set by page creator
reviewer: Kimberly Horan # to be set by the page reviewer
reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
```

File: content/orm-part-1/reading/walkthrough/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+Welcome to the walkthrough for creating and using templates in your CodingEvents project.
+
+You will find a video walkthrough of the steps covered in the reading. Time stamps have been added to highlight the topics covered so far and their locations in the reading.
+
+## Video Walkthrough
+
+{{<youtube mza9QdrCv2U>}}
```
Comment (codinglikeagirl42) [inline content/orm-part-1/reading/walkthrough/index.md:20]: Please add chapters to the video.

**Review by codinglikeagirl42 — APPROVED** [review]
Looks good, few minor suggestions, then Merge Away!

File: content/orm-part-1/reading/walkthrough/index.md (hunk 3 above)
Comment (speudusa) [reply content/orm-part-1/reading/walkthrough/index.md:20]: chapters have been added. :)

**Pull request merged**